ras_party/controllers/pending_survey_controller.py

```diff
@@ -404,8 +406,9 @@ def send_pending_surveys_confirmation_email(pending_surveys_list, confirmation_e
         else:
             confirmation_email_template = 'share_survey_access_confirmation'
         business_list = []
-        for survey in pending_surveys_list:
-            business = get_business_by_id(str(survey['business_id']))
+        business_id_list = [pending_surveys['business_id'] for pending_surveys in pending_surveys_list]
```

Comment:
If you change this to:
```python
business_id_list = {pending_surveys['business_id'] for pending_surveys in pending_surveys_list}
```
Doing a set comprehension will let you do the following on the next line.  It's minor but it's tidier.

```python
for business_id in business_id_list
```